code/processes/rdb.q

```diff
@@ -235,3 +235,6 @@ $[.rdb.connectonstart;
 
 //Updating .rdb.subtables to only include tables that actually subscribed
 .rdb.subtables:$[.ds.datastripe;tables[] except `heartbeat;.rdb.subtables];
+
+/- initialise datastripe
+if[.ds.datastripe;initdatastripe[];.lg.o[`dsinit;"datastripe on: initialising datastripe"]];
```

Comment:
minor nitpick - you're logged _after_ calling initdatastripe, so should the log be `initialised datastripe` instead? or come before the `initdatastripe[]` call?

Reply:
Yes sorry, log should be before